package.json

```diff
@@ -148,7 +148,8 @@
       "jsx",
       "json",
       "node"
-    ]
+    ],
+    "setupTestFrameworkScriptFile": "<rootDir>/src/components/banner/MockLocalStorage.tsx"
```

Comment:
According to jest documentation we should be using setupFilesAfterEnv in new code

Reply:
Link? When I use that variable name, I get the following warning: 
"Validation Warning: Unknown option "setupFilesAfterEnv" with value <rootDir>/src/components/banner/MockLocalStorage.tsx was found. This is probably a typing mistake. Fixing it will remove this message."

Reply:
https://jestjs.io/docs/en/configuration#setupfilesafterenv-array

Reply:
Hmm, the tests do still pass despite that warning so I guess it's ok?

Reply:
Ah the problem is that we're using jest v23, but that field is added in v24. You can do npm install jest@latest --save to update it, but it looks like this is actually not needed because node.js actually has some implementation of local storage - you can log localStorage.getItem.toString() in a test to see what that is. So I think this can just be deleted